test/verify/machineslib.py

```diff
@@ -1859,7 +1859,7 @@ def tryCreate(self, dialog):
                     b.wait_in_text("#vm-{0}-disks-vda-device".format(name), "disk")
                 else:
                     b.wait_in_text("#vm-{0}-disks-hda-device".format(name), "disk")
-            elif (dialog.storage_pool == 'No Storage' and dialog.sourceType == 'file') or (dialog.sourceType == 'url' and dialog.start_vm):
+            elif dialog.start_vm and (((dialog.storage_pool == 'No Storage' or dialog.storage_size == 0) and dialog.sourceType == 'file') or dialog.sourceType == 'url'):
```

Comment:
This logically seems to be a fixup of the previous commit. If it is, we can squash it on merging, and add the "Closes #"  to the commit message then.

If it's an independent change, please give some explanation in the commit message, and add a "Closes #12063" to the commit message. PRs with more than one commit need to have these so that we can land them with "Rebase & Merge".

Reply:
I kinda reworked the condition so it's more accurate. Not exactly a fixup, but I can tell where I should have done things differently:
- 9b5ed7da4cfbab059a94de72c62576088b09b021 (I should have added start_vm for whole condition)
- a063c8bf0f5a36ba7720554e6be57cbb711544df (I should have added storage_size == 0)
both commits are in different PRs.

Should I therefore split this commit into two commits, and each will close it's relative PR where change should have been made?